### ISSUE TITLE
Segfault (nil reference) in templates.go

### ISSUE THREAD

**Issue by RapidRotator** [writefreely/writefreely#700]
When I start writefreely, it fails after this:
```
2023/05/26 06:34:09 Loading templates...
2023/05/26 06:34:09 Loading pages...
2023/05/26 06:34:09 Loading user pages...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1253bde]

goroutine 1 [running]:
github.com/writefreely/writefreely.InitTemplates.func2({0xc0001a0de0, 0x13}, {0x0, 0x0}, {0x14?, 0x3?})
        /src/writefreely/src/writefreely-0.13.2/templates.go:150 +0x3e
path/filepath.Walk({0xc0001a0de0, 0x13}, 0xc000595b70)
        /usr/lib/go/src/path/filepath/path.go:569 +0x50
github.com/writefreely/writefreely.InitTemplates(0xc000351200)
        /src/writefreely/src/writefreely-0.13.2/templates.go:149 +0x314
github.com/writefreely/writefreely.Initialize({0x16c1580, 0xc00002e180}, 0x5?)
        /src/writefreely/src/writefreely-0.13.2/app.go:394 +0x4b
main.legacyActions(0xc0003bdb80?)
        /src/writefreely/src/writefreely-0.13.2/cmd/writefreely/main.go:161 +0x2fc
github.com/urfave/cli/v2.(*App).RunContext(0xc00008bd40, {0x16c0a28?, 0xc00011a000}, {0xc000112210, 0x1, 0x1})
        /home/builduser/go/pkg/mod/github.com/urfave/cli/v2@v2.5.1/app.go:322 +0x97c
github.com/urfave/cli/v2.(*App).Run(...)
        /home/builduser/go/pkg/mod/github.com/urfave/cli/v2@v2.5.1/app.go:224
main.main()
        /src/writefreely/src/writefreely-0.13.2/cmd/writefreely/main.go:118 +0x7d6
```
This is a new installation in a docker container where I successfully completed the other steps for configuration and key generation. It seems to complain about templates.

Is there a default of example template available that I can use?

### Steps to reproduce
1. Start a docker container based on Archlinux with Writefreely and its dependencies installed
2. Run `writefreely config start`
3. Run `writefreely keys generate`
4. Run `writefreely`

### Expected behavior
Writefreely to either run or clearly indicate what it is missing.

### Application configuration
 - **Single mode
 - **Database? [mysql]
 - **Open registration?[no]
 - **Federation enabled? [yes]

**Version or last commit**:
0.13.2

**Comment by thebaer** [comment]
As mentioned in [this comment](https://github.com/writefreely/writefreely/issues/675#issuecomment-1563883336), this is probably due to an uncaught error when walking the `templates/user/` directory. The bug we'll need to fix is: catching any errors and displaying a friendlier message.

But to fix underlying cause, you'll want to make sure your `templates_parent_dir` config value is set correctly to an existing `templates/` directory, and that the application can reach it.

**Comment by RapidRotator** [comment]
I have, and this solves it, but now I'm facing these:
```
ERROR: 2023/05/26 07:55:51 collections.go:873: Unable to render collection index: template: collection.tmpl:9:7: executing "collection" at <.CustomCSS>: can't evaluate field CustomCSS in type writefreely.CollectionPage
ERROR: 2023/05/26 07:55:51 handle.go:533: [Web handler] 500: template: collection.tmpl:9:7: executing "collection" at <.CustomCSS>: can't evaluate field CustomCSS in type writefreely.CollectionPage
2023/05/26 07:55:51 Web internal error render
2023/05/26 07:55:51 "GET /" 500 28.99877ms "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/113.0"
2023/05/26 07:55:51 http: superfluous response.WriteHeader call from github.com/writeas/impart.writeBody (response.go:31)
```

**Comment by thebaer** [comment]
This is fixed (I believe) by #621, available in latest version 0.14.0
